pytorch_lightning/accelerators/tpu.py

```diff
@@ -59,19 +59,16 @@ def clip_gradients(
         self,
         optimizer: Optimizer,
         clip_val: Union[float, int],
-        norm_type: float = 2.0,
```

Comment:
this removed on purpose?

Reply:
Yes, the parent's signature does not have this argument so mypy complains

Also, there is currently no way for users to change the `norm_type` value so it's okay to remove